Cubical/Algebra/Algebra/Base.agda

```diff
@@ -178,6 +195,19 @@ isPropIsAlgebra R _ _ _ _ _ _ =
   where
   open IsLeftModule
 
+isPropIsAlgebraHom : (R : Ring ℓ) {A : Type ℓ'} {B : Type ℓ''}
+                     (AS : AlgebraStr R A) (f : A → B) (BS : AlgebraStr R B)
+                   → isProp (IsAlgebraHom AS f BS)
+isPropIsAlgebraHom R AS f BS =
+  isOfHLevelRetractFromIso 1 IsAlgebraHomIsoΣ
+   (isProp× (isSetAlgebra (_ , BS) _ _)
+     (isProp× (isSetAlgebra (_ , BS) _ _)
+       (isProp× (isPropΠ2 λ _ _ → isSetAlgebra (_ , BS) _ _)
+         (isProp× (isPropΠ2 λ _ _ → isSetAlgebra (_ , BS) _ _)
+           (isProp× (isPropΠ λ _ → isSetAlgebra (_ , BS) _ _)
```

Comment:
Can't you use some of the higher arity `isProp×`?

Reply:
Yeah that's much nicer. I basically copied this proof from Ring.Base, so I fixed it there as well.